mlrun/api/initial_data.py

```diff
@@ -57,16 +62,10 @@ def _is_latest_data_version():
     db_session = create_session()
     db = mlrun.api.db.sqldb.db.SQLDB("")
 
-    try:
-        current_data_version = int(db.get_current_data_version(db_session))
-        return current_data_version == latest_data_version
-    except Exception as exc:
-        logger.info(
-            "Failed getting current data version, assuming old version", exc=exc
-        )
-        return False
-    finally:
-        close_session(db_session)
+    current_data_version = _get_current_data_version(db, db_session)
+    close_session(db_session)
```

Comment:
I think it would still be safer to wrap the first line with try and close the session in the finally no ?